src/index.js

```diff
@@ -31,13 +31,22 @@ const searchFromQuerystring = (solrClient, options = {}) => {
       // Set the state of the main query field to the value of the search qs param
       searchField.value = parsed.search;
     }
-
     // If the searchField is one for which we preserve state through qs.
     if (helpers.filterFieldsWithQsState.find((filterField) => filterField === searchField.field )) {
       searchField = helpers.qs.setFieldStateFromQs({
         params,
         searchField
-      })
+      }) // this resets our initial state of search sites
+    }
+    // Account for default site search configuration, if present and no site
+    // has been selected.
+    if (searchField.field === 'sm_site_name' && searchField.value === undefined && options.siteList.length > 0) {
+      searchField.value = options.siteList;
+    }
+    // If restricted to the current site by configuration, enforce it here.
+    // @TODO: There doesn't seem an obvious way to pass this value.
```

Comment:
@jesconstantine I can't find anywhere in the current code where the `siteSearch` variable from the variables is actually used to enforce the setting for "only search this site by default."